oxide/lib.go

```diff
@@ -55,6 +72,11 @@ type Client struct {
 	userAgent string
 }
 
+type hostCreds struct {
+	host  string
+	token string
+}
+
 // NewClient creates a new client for the Oxide API. To authenticate with
 // environment variables, set OXIDE_HOST and OXIDE_TOKEN accordingly. Pass in a
 // non-nil *Config to set the various configuration options on a Client. When
```

Comment:
Can we add some information about the new functionality here?

Reply:
Whoops, good catch! Added.